docs/upctl.md

```diff
@@ -0,0 +1,35 @@
+## upctl
+
+UpCloud CLI
+
+### Synopsis
+
+upctl a CLI tool for managing your UpCloud services.
+
+```
+upctl [flags]
+```
+
+### Options
+
+```
+  -t, --client-timeout duration   CLI timeout when using interactive mode on some commands (default 1m0s)
+      --colours                   Use terminal colours (supported: auto, true, false) (default true)
+      --config string             Config file
+  -h, --help                      help for upctl
+  -o, --output string             Output format (supported: json, yaml and human) (default "human")
+  -w, --wait                      Wait for the command to be completed
```

Comment:
looks like wait snuck back in from somewhere?

Reply:
The branch was just behind & the docs were generated from older code, now up-to-date :)